.github/workflows/pr.yml

```diff
@@ -55,9 +63,11 @@ jobs:
       - name: Install dependencies and build
         env:
           DEFI_SDK_API_URL: ${{ github.event.inputs.DEFI_SDK_API_URL || 'wss://api-v4.zerion.io/' }}
+          DEFI_SDK_TESTNET_API_URL: ${{ github.event.inputs.DEFI_SDK_TESTNET_API_URL || 'wss://api-testnet.zerion.io/' }}
           ZERION_API_URL: ${{ github.event.inputs.ZERION_API_URL || 'https://zpi.zerion.io/' }}
+          ZERION_TESTNET_API_URL: ${{ github.event.inputs.ZERION_TESTNET_API_URL || 'https://zpi-testnet.zerion.io/' }}
           BACKEND_ENV: ${{ github.event.inputs.BACKEND_ENV || '' }}
-          FEATURE_PAYMASTER: ${{ github.event.inputs.FEATURE_PAYMASTER || '' }}
+          FEATURE_PAYMASTER: ${{ github.event.inputs.FEATURE_PAYMASTER || 'on' }}
```

Comment:
Looks like there should be `FEATURE_PAYMASTER ` in the inputs section in the beginning of the config

Reply:
I think it's there, right? https://github.com/zeriontech/zerion-wallet-extension/blob/main/.github/workflows/pr.yml#L12-L15